kiota_http/httpx_request_adapter.py

```diff
@@ -189,7 +200,7 @@ async def send_collection_async(
         self,
         request_info: RequestInformation,
         parsable_factory: ParsableFactory,
-        error_map: Dict[str, ParsableFactory],
+        error_map: Dict[str, ParsableFactory] = {},
```

Comment:
would we need to update the interface as well if we made this optional?
also why would we make that change only for this method?

Reply:
Good catch. This was for local testing purposes and meant to be removed

Reply:
And no, setting a default value does not require an update to the interface. It's merely an extension